src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -1618,7 +1618,21 @@ void enableSSL(String host,
 
                 tm = new TrustManager[] {new PermissiveX509TrustManager(this)};
             }
-
+            // Otherwise, we'll check if a specific TrustManager implemenation has been requested and
+            // if so instantiate it, optionally specifying a constructor argument to customize it.
+            else if (con.getTrustManagerClass() != null)
+            {
+                Class<?> tmClazz = Class.forName(con.getTrustManagerClass());
+                if (!TrustManager.class.isAssignableFrom(tmClazz)) {
+                    throw new IllegalArgumentException("The class specified by the trustManagerClass property must implement javax.net.ssl.TrustManager");
```

Comment:
Please also use the Microsoft code [formatter ](https://github.com/Microsoft/mssql-jdbc/blob/master/mssql-jdbc_formatter.xml)for the changes.